Multisig.xcodeproj/project.pbxproj

```diff
@@ -928,7 +928,7 @@
 					"@executable_path/Frameworks",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = io.gnosis.multisig.prerelease;
-				PRODUCT_NAME = "Safe Multisig";
+				PRODUCT_NAME = Multisig;
```

Comment:
@sche what's that? why not Safe Multisig ? 😄

Reply:
then in test target, I need to import Safe_Multisig. So I returned it to how it was (as I introduced Safe Multisig in 0.1.0 release) so we could decide later together on the name.